guides/v2.1/extension-dev-guide/versioning/check-version.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Check the Magento version
+---
+
+There are the next ways how to get a version of Magento store:
+ - through command line
+ - through HTTP GET request
+ - from Magento backend (admin account is required)
+ - from `composer.lock` file
+ 
+#### Command line
+The command below returns a full Magento version.
+
+**Command**
+```text  
+php bin/magento --version
+```
+
+**Response**
+```text
+Magento CLI version 2.3.0
+```
+
+#### HTTP GET request
+HTTP request allows to get a short information about Magento version.
+
+**Request**
+```text
+http://<magento2-store>/magento_version
+```
+
+**Response**
+```text
+Magento/2.3 (Community)
+```
+
+#### Magento backend
+
+Login as a registered admin user into back-end side of your Magento store. Then go to the bottom of the page. Magento version will be displayed in the right corner above "Account Activity" and "Report an Issue" links:
+
+![Check the Magento version]({{ page.baseurl }}/extension-dev-guide/images/version.png)
+
+#### File `composer.lock`
+Please note that Magento reads current version from the `composer.lock` file that locates in the root of the store. If you want to get Magento version from this file then make search for `magento/product-community-edition` (if you use community edition) or `magento/product-enterprise-edition` (if you use commerce version):
```

Comment:
```suggestion
Magento reads the current version from the `<Magento_root>/composer.lock` file. To determine the Magento version, search for `magento/product-community-edition` or `magento/product-enterprise-edition`, depending on which version of Magento is installed.
```

Reply:
Please verify this statement. I don't see any reference to `magento/product` here:
https://github.com/magento/magento2/blob/2.3-develop/composer.lock

Reply:
> Please verify this statement. I don't see any reference to magento/product here:
https://github.com/magento/magento2/blob/2.3-develop/composer.lock

I've verified this step within debug.

Reply:
@keharper , all changes have been implemented. Could you please review? 🎉 

Thank you!